SpatialGDK/Documentation/content/get-started/manual-engine-build.md

```diff
@@ -17,12 +17,7 @@ You must have followed the [Get started]({{urlRoot}}/content/get-started/introdu
 * [1 - Get the dependencies]({{urlRoot}}/content/get-started/dependencies)
 * [2 - Set up the fork and plugin]({{urlRoot}}/content/get-started/build-unreal-fork.md) up to [step 5]({{urlRoot}}/content/get-started/build-unreal-fork#step-5-clone-and-install-the-plugin)
 
-## Step 1: Unreal Linux cross-platform support</br>
-
-To build the server software for SpatialOS deployments correctly, you need to build the Unreal Engine fork targeting Linux. This requires Linux cross-compilation of your SpatialOS project and the Unreal Engine fork. To do this, you need to download and unzip the Linux cross-compilation toolchain.</br></br>
-For guidance on this, see the _Getting the toolchain_ section of Unreal's [Compiling for Linux](https://wiki.unrealengine.com/Compiling_For_Linux) documentation. As you follow the guidance there, select **v11 clang 5.0.0-based** to download the `v11_clang-5.0.0-centos7.zip` archive, then unzip this file into a suitable directory.
-
-## Step 2: Add a LINUX_MULTIARCH_ROOT environment variable
+## Step 1: Add a LINUX_MULTIARCH_ROOT environment variable
```

Comment:
I believe the variable is set automatically by running the toolchain installer.

Reply:
That is correct

Reply:
PTAL now